23-07-workshop/finished/Leavitts_Footsteps_solutions.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<p>aha. the reason why some of these are normalized and others are not is that the "authors" are different. the QLP (quick-look pipeline) generates normalized light curves; there is no "raw flux" to fall back on.</p><p> </p><p>Ideally, for consistency of data reduction techniques, we would pick one TESS pipeline for all of these light curves. Unfortunately, there are only two "authors" in common for this set of TIC IDs: 'GSFC-ELEANOR-LITE' and 'TGLC'. TGLC isn't supported by lightkurve, and the GSFC curves are pretty "raw", in the sense that they have nonsense values like negative flux</p><p> </p><p>potential solutions</p><ol><li>leave it "as-is" and add a disclaimer about the different pipelines</li><li>use GSFC and do a "quick and dirty" sigma clip (it produces decent results with<code>.remove_outliers(5)</code>)</li><li>other solutions at your discretion</li></ol><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
This just in! Lightkurve 2.4.1 now supports TGLC, so we could actually use this

Reply:
<p>Great, I changed the instructions to use TGLC</p>